.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,31 @@
+## Motivation
+
+Closes [BZ#NNNN](https://coq.inria.fr/bug/NNNN). <!-- If applicable, otherwise describe the issue if it is not obvious. -->
```

Comment:
outdated ;)

Reply:
Indeed, thanks @SkySkimmer.
However, I won't fix it right away because this PR is also pending on the two questions I asked in my comment above: what strategy do we adopt for `CHANGES` and what is our compatibility policy.